frontend/src/store/objectStore.ts

```diff
@@ -208,6 +208,8 @@ export const useObjectStore = defineStore('object', () => {
     try {
       appStore.beginIndeterminateLoading();
       await objectService.togglePublic(objectId, isPublic);
+      const obj = findObjectById(objectId);
+      if (obj) obj.public = isPublic;
```

Comment:
Are these two new lines doing anything?
was the intention to update the store. eg: `if (obj) state.objects.value.find((x) => x.id === objectId).public = isPublic`

Reply:
Yes the store needs to be updated with the new value. Would prefer to leave this using the findObjectById function in the rare event we change how we are looking up objects. 
`findObjectById(objectId)!.public = isPublic` should work.

Reply:
Yes we need to update the store here so the new value is reflected elsewhere. Would prefer to leave this using findObjectById() in the event we change up how we are looking up objects.

Reply:
oh ok. i didnt know you could update the store like this